### PR TITLE
Implement get() for register endpoint.

### PR DESCRIPTION
**Pull request by toolgirl**
From @toolgirl and @brsaylor

This endpoint returns a JSON object equivalent to legacy_lark_config.json with the exception of pricing logic which is now broken into the registration and camper level components.

### CODE REVIEW

File: server/camphoric/models.py [hunk 1]
```diff
@@ -81,21 +85,24 @@ def __str__(self):
...
-    - Should be able to track capacity so that lodging options are removed from registration as they fill up
+    - Should be able to track capacity so that lodging options are
+        removed from registration as they fill up
     '''
     event = models.ForeignKey(Event, on_delete=models.CASCADE)
     parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True)
     name = models.CharField(max_length=255)
     capacity = models.IntegerField(default=0)
     notes = models.TextField()
+    # is_visible = models.boolean()
```
Comment (evinism) [inline server/camphoric/models.py]: presumably remove?

File: server/camphoric/models.py [hunk 2]
```diff
@@ -57,7 +57,11 @@ class Event(TimeStampedModel):
     camper_schema = JSONField(null=True)
     payment_schema = JSONField(null=True)
     registration_schema = JSONField(null=True)
-    lodging_schema = JSONField(null=True)
+    registration_ui_schema = JSONField(null=True)
```
Comment (evinism) [inline server/camphoric/models.py]: I think these fields should probably have some comments describing what they mean semantically -- it's not clear what each of these schemas represent and why they differ from one another.

Comment (brsaylor) [reply server/camphoric/models.py]: Good thought - done.

File: server/camphoric/models.py (hunk 1 above)
Comment (brsaylor) [reply server/camphoric/models.py]: Sounds good. Removed and created #40.

**Pull request merged**